src/main/java/net/openhft/chronicle/queue/impl/single/StoreTailer.java

```diff
@@ -195,6 +201,25 @@ public DocumentContext readingDocument(final boolean includeMetaData) {
         return documentContext;
```

Comment:
Do we need to replicate this check in the named version? or can we remove it from named and unnamed? it makes it more confusing if they're inconsistent